src/components/components/ebay-dialog-base/component.js

```diff
@@ -68,13 +84,31 @@ module.exports = {
         });
     },
 
+    triggerFocus(focusEl) {
```

Comment:
I'd prefer these new methods use `_` to be consistent with what we've done for the other internal methods.